tests/test_summary.py

```diff
@@ -33,6 +33,12 @@
 TESTDIR = Path(__file__).absolute().parent
 REEK = str(TESTDIR / "data/reek/eclipse/model/2_R001_REEK-0.DATA")
 EIGHTCELLS = str(TESTDIR / "data/eightcells/EIGHTCELLS.DATA")
+SHORT_STEP = str(TESTDIR / "data/timesteps/SHORT_STEP")
```

Comment:
Have these been made with EIGTHCELLS? Could these example UNSMRY be made reproducible?

Reply:
Was made with the reek model, but doesn't really matter. Only thing was to have a schedule with forced steps, and as stripped UNSMRY files as possible. In the scenario without TIMESTEP: no vectors where requested in the SUMMARY section. In the scenarios with TIMESTEP: TIMESTEP was the only requested vector in the SUMMARY section. If I am to make it reproducible, I guess the EIGHTCELLS is better to make it light